test/core/integration/Integration.t.sol

```diff
@@ -67,52 +91,4 @@ abstract contract Integration_Test is Base_Test {
         assertFalse(success, "delegatecall success");
         assertEq(returnData, abi.encodeWithSelector(Errors.DelegateCall.selector), "delegatecall return data");
     }
-
-    /*//////////////////////////////////////////////////////////////////////////
-                                      HELPERS
-    //////////////////////////////////////////////////////////////////////////*/
-
-    /// @dev Creates the default stream.
-    function createDefaultStream() internal virtual returns (uint256 streamId);
```

Comment:
OK looks good. But what about other create functions? Do we keep them here as well? I'd also like to pick your brain on https://github.com/sablier-labs/flow/pull/334. 

Where `Integration` contains common variables and methods across concrete and fuzz whereas Concrete and Fuzz contain variables and methods to be required in Concrete and Fuzz tests respectively. Make it more organized. What do you think?

```mermaid
flowchart TD;
  Base --> Integration
  Integration --> Concrete
  Integration --> Fuzz
  Base --> Fork
```

If we follow this approach, then it should not be placed in Integration if they are not used in Fuzz tests.

Reply:
Its not related to Flow. Its related to V2-core as well.

Reply:
> Its not related to Flow

i know, but what i meant is: let's first finish with the current changes, and then consider this change (trust me it, it would be a bigger change than in flow)